graphql/database/experiments/features.js

```diff
@@ -43,30 +43,60 @@ const features = {
 
 features[YAHOO_SEARCH_EXISTING_USERS] = {
   defaultValue: false,
-  // Enable when SFAC search engine is enabled.
-  // rules: [
-  //   {
-  //     condition: {
-  //       isTabTeamMember: true,
-  //       env: 'local',
-  //     },
-  //     force: true,
-  //   },
-  // ],
+  rules: [
+    /* Begin internal overrides */
+    {
+      condition: {
+        v4BetaEnabled: true,
+        'internalExperimentOverrides.search-existing-users': {
+          $eq: true,
+          $exists: true,
+        },
+      },
+      force: true,
+    },
+    {
+      condition: {
+        v4BetaEnabled: true,
+        'internalExperimentOverrides.search-existing-users-v2': {
```

Comment:
Should this be the same key as the above? FYI, in case you didn't know, you can use variables in key names like:
```js
{
  [`my-string-and-${someVar}`]: "foo",
}
```

Reply:
Good call on the key being the same.